scout/server/blueprints/variants/forms.py

```diff
@@ -41,3 +41,22 @@ class FiltersForm(FlaskForm):
 
     thousand_genomes_frequency = DecimalField('1000 Genomes', places=2)
     exac_frequency = DecimalField('ExAC', places=2)
+
+class SvFiltersForm(FlaskForm):
```

Comment:
Maybe this class can inherit the common fields from another class?

We will look into it after merging this PR 👍